windows-agent/internal/proservices/landscape/executor_test.go

```diff
@@ -142,21 +146,46 @@ func TestReceiveCommandStartStop(t *testing.T) {
 func TestInstall(t *testing.T) {
 	t.Parallel()
 
+	ctx, cancel := context.WithCancel(context.Background())
+	t.Cleanup(cancel)
+
+	fileServerAddr := mockRootfsFileServer(t, ctx)
+
+	emptyFileChecksum := "e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855"
+	mockErrorChecksum := "afe55cda4210c2439b47c62c01039027522f7ed4abdb113972b3030b3359532a"
+	mockMismatchChecksum := "1234"
+
 	testCases := map[string]struct {
-		distroAlredyInstalled bool
-		emptyDistroName       bool
-		wslInstallErr         bool
-		appxDoesNotExist      bool
+		distroAlreadyInstalled bool
+		emptyDistroName        bool
+		wslInstallErr          bool
+		appxDoesNotExist       bool
+		// to improve testing coverage
+		nonResponsiveServer  bool
+		breakVhdxDir         bool
+		breakTarballCreation bool
+
+		sendRootfsURL      string
+		sendRootfsChecksum string
 
 		wantInstalled   bool
 		wantNonRootUser bool
```

Comment:
I see this one is only use when downloading from the Store, I’m a little bit puzzled in its usage, mind looking at it?

Reply:
With the other tests it still seems like they have a nonroot user anyway, so I'm not sure.

Reply:
I guess we could remove the assertion about whether the default user is root or not, since it's only relevant in one test case, which is related to installing from the MS Store.